AWS-ECS-AutoStopping-Tag-Support (#366)
harness/developer-hub: docs/cloud-cost-management/2-use-cloud-cost-management/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/3-create-auto-stopping-rules/create-auto-stopping-rules-for-ecs.md

1. In **Select the resources to be managed by the rule**, select **ECS Service**, and then click **Add an ECS Service**.
   
     ![](./static/create-auto-stopping-rules-for-ecs-05.png)
2. In **Select ECS Service**, select one of the following options:

>    * ECS Service Name
>    * ECS Service Tags

Reviewer: ```suggestion
   * ECS Service Name.
   * ECS Service Tags.
```

Author: Done

>   If you have chosen **ECS Service Name**, select a region and a cluster to see all the services:
>   1. Select the region where your cluster is hosted from the **Select Region** dropdown list.
>   2. Select your cluster from the **Select Cluster** dropdown list.
>   3. Select the ECS Service for which you want to enable AutoStopping Rule. 
>   4. Click **Add Selected**.

Reviewer: ```suggestion
  If you chose **ECS Service Name**, select a region and a cluster to see all of their services:
  1. Select the region where your cluster is hosted from the **Select Region** dropdown list.
  2. Select your cluster from the **Select Cluster** list.
  3. Select the ECS Service where you want to enable the AutoStopping Rule. 
  4. Click **Add Selected**.
```

Author: Done

   
   ![](./static/aws-ecs-tags.png)
   

> If you have chosen **ECS Service Tags**, select the region, cluster, and the tag associated with the service. Once you create a rule using this option, it is applied automatically to the most recently created ECS service to which the tag is added. 

Reviewer: ```suggestion
If you have chose **ECS Service Tags**, select the region, cluster, and the tag associated with the service. Once you create a rule using this option, the rule is applied automatically to the most recently created ECS service using the tag. 
```

Author: Done

1. Select the region where your cluster is hosted from the **Select Region** dropdown list.
2. Select your cluster from the **Select Cluster** dropdown list.
3. Select the tag key and the tag value from the dropdown lists. If you want to learn how to add tags to your service, go to [Tagging your Amazon ECS Resources](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-using-tags.html).
   

>      You can view the list of ECS services to which this tag is added. 

Reviewer: ```suggestion
     You can view the list of ECS services using this tag. 
```

Author: Done

4. Click **Continue**.  

:::note
* If you have added the same tag to multiple ECS services, the AutoStopping rule is applied to the most recently created service.
* If there are no services with the selected tag, the following error message is displayed: 
  "There is no ECS service that contains the selected tag. Create a service with the selected tag to proceed."
:::

![](./static/aws-ecs-use-tags.png)

   
### **Specify the desired task count**
In **Desired Task Count**, specify the desired task count for the selected ECS service. This is the number of tasks that Harness will instantiate when your service is up and running. For more information, see [Amazon ECS service quotas](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/service-quotas.html).

> ### (Optional) Set Up Advanced Configuration

Reviewer: ```suggestion
### (Optional) Set up advanced configuration
```

Author: Done
